homework-submission/homework-week-4/src/App.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './App.css';
+import Root from './components/root-component/root';
+import Week1 from './components/week1-component/static-todo-list';
+import Week2 from './components/week2-component/dynamic-todo-list';
+import Week3 from './components/week3-component/people';
+import { Route, NavLink, BrowserRouter } from 'react-router-dom';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <React.Fragment>
```

Comment:
React Fragments is used to group items without added an extra node, but you added a `<div>` inside the Fragment rendering it useless. Since `<BrowserRouter>` is a parent in this case, you don't need `<React.Fragment>` and the `<div>`, you can delete both of them!

Reply:
More on Fragments; https://reactjs.org/docs/fragments.html

Reply:
Good to be learned that.
<React.Fragment> tags has been deleted.